index.bs

```diff
@@ -2277,6 +2285,10 @@ Start of a Review Period</h4>
 	Each Member organization <em class="rfc2119">may</em> send one review,
 	which <em class="rfc2119">must</em> be returned by its [=Advisory Committee representative=].
 
+	For clarity,
+	in the context of an [=AC Review=],
+	[=dissent=] <em class=rfc2119>must</em> be expressed as a [=Formal Objection=].
```

Comment:
```suggestion
	[=dissent=] <em class=rfc2119>may</em> be expressed as a [=Formal Objection=].
```

There are options in the standard review forms we use to say "I disagree but ths is **not** a formal objection", which I think is appropriate.

Reply:
Yes, it is, but that's not dissent in our formal definition, it's mere disagreement. Dissent is the presence of one or more FOs.

Reply:
I'm confused, maybe due to the difficult of seeing the preview. I read "A [=Formal Objection=] always indicates sustained disagreement,	but isn't necessary to express [=dissent=]." So, is dissent a sign of the presence of one or more FOs?

Reply:
In some contexts, Dissent can only be expressed as an FO (AC Review). But in a WG, if the chair is trying to assess consensus, they can ask the something vaguely like "do we have any sustained objections to this proposed decision?" and if there is, they shouldn't (can't?) declare consensus. The member doesn't need (shouldn't) file an FO from the room in a hurry.

Reply:
> "A [=Formal Objection=] always indicates sustained disagreement, but isn't necessary to express [=dissent=]."

The presence of "sustained disagreement" and "dissent" are the same, so using different words here might have been confusing. I think we should rephrase as

> "A [=Formal Objection=] always indicates sustained disagreement, but isn't necessary to express it."


"sustained objection" = "I insist I cannot agree to this, and therefore, because of that disagreement, we do not have consensus."

"Formal objection" = "sustained objection + I want kick off the council process"

(Note: As per https://www.w3.org/2021/Process-20211102/#managing-dissent, even if there is not consensus, the chair can still record decisions (possibly by taking a vote), so we're not stuck.)

Reply:
yes. "isn't necessary to express it in all circumstances"? or flip it and "and is the only way to express sustained disagreement in some (formal) contexts, such as AC votes"?

Reply:
I agree with taking @dwsinger's suggested rephrasing.